src/components/incidents/IncidentFilter.js

```diff
@@ -0,0 +1,40 @@
+import { DateTime } from 'luxon';
+import Interval from 'luxon/src/interval.js';
+
+const newData = arr => {
```

Comment:
While it's obvious that "arr" is an array, I think it's valuable to use descriptive and declarative names with your variables.

Reply:
All of this were left over from Anthony's code, I'm not sure what he's doing with this function. I'll get to altering variables to make this understandable